app/frontend/src/components/TextField.tsx

```diff
@@ -24,6 +24,9 @@ const useStyles = makeStyles((theme) => ({
 interface AccessibleTextFieldProps extends BaseTextFieldProps {
   id: BaseTextFieldProps["id"];
   onChange?: TextFieldProps["onChange"];
+  fullWidth?: TextFieldProps["fullWidth"];
```

Comment:
Why are these needed? Aren't they in `BaseTextFieldProps` already?

Reply:
Yes, you're right!